submissions/BogdanGapon/HTML & CSS popup/sass/layout/_header.scss

```diff
@@ -0,0 +1,176 @@
+.header {
+  padding:20px;
+  background-color:#fff;
+}
+
+.header-nav__list {
+  display:flex;
+  justify-content:flex-end;
+  align-items:center;
+  gap:25px;
+}
+
+.header-nav__item {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  flex-basis:30px;
+}
+
+.header-nav__link {
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  text-decoration:none;
+  color:rgb(17, 15, 15);
+  transition:250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+
+  &:hover,
+  &:focus {
+    color:rgb(170, 63, 63);
+  }
+
+  &.link-img:hover {
+    box-shadow:2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+  }
+}
+
+.visually-hidden {
+  position:absolute;
+  appearance:none;
+}
+
+.custom-checkbox {
+  position:relative;
+
+  &::before {
+    content:' ';
+    display:inline-flex;
+    justify-content:center;
+    align-items:center;
+    width:30px;
+    height:30px;
+    background-image:url(../img/popup-button.png);
+    background-repeat:no-repeat;
+    background-size:30px 30px;
+    cursor:pointer;
+  }
+}
+
+.visually-hidden:focus ~ .custom-checkbox::before {
+  outline:2px solid black;
+}
+
+.popup-wrapper {
+  display:none;
+  position:absolute;
+  top:65px;
+  right:50px;
+  width:350px;
+  height:400px;
+  padding:30px 15px 30px;
+  border:1px solid rgba(128, 128, 128, 0.4);
+  box-shadow:1px 1px 2px rgba(128, 128, 128, 0.4);
+  background-color:#fff;
+  overflow:auto;
+
+  .checkbox:checked ~ & {
+    display:block;
+  }
+}
+
+.popup-item {
+  display:flex;
+  flex-wrap:wrap;
+  justify-content:center;
+  align-items:center;
+  margin-bottom:10px;
+  gap:40px;
+}
+
+.popup-wrapper img {
+  margin-bottom:10px;
+}
+
+.span {
+  content:' ';
+  position:absolute;
+  top:55px;
+  z-index:10;
+
+  display:none;
+  width:20px;
+  height:20px;
+  border-top:1px solid rgba(128, 128, 128, 0.4);
+  border-left:1px solid rgba(128, 128, 128, 0.4);
+  transform:rotate(45deg);
+  background-color: #fff;
+
+  .checkbox:checked + & {
+    display: block;
+  }
+}
+
+.popup-list__list {
+  width:65px;
+  display:flex;
+  justify-content:center;
+  align-items:center;
+
+  &:hover,
+  &:focus {
+    border: 1px solid grey;
+  }
+
+  & a {
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    justify-content:center;
+    text-decoration:none;
+    color:#000;
+  }
+}
+
+img {
+  display:block;
+  max-width:100%;
+  object-fit:contain;
+}
+
+.label-more {
+  position:absolute;
+  display:flex;
+  justify-content:center;
+  align-items:center;
+  width:100%;
+  height:40px;
+  left:0;
+  z-index:1;
+  background-color:bisque;
+  cursor:pointer;
+
+  .checkbox-more:focus ~ & {
+    border:1px solid #000;
+  }
+
+}
+
+.next-popup-wrapper {
+  display:none;
```

Comment:
Why all your css rules don't have space after `:`?
` display: none;`
Maybe, formatter issue?
Can you fix it?

Reply:
Just say that you seriously ??  You write Please, fix spacing i DELETE all spacing after css :,i mean i sit and delete all spaces which you say and now you say not delete? I don`t understund what i need to do, you write more 10 comments with "Please, fix spacing", i fixed all spacing and now i need to remote it again??

Reply:
![Screenshot_2](https://user-images.githubusercontent.com/95972709/181197489-1f9c5ae4-a430-4f0a-b2c5-4f65eddf42e8.png)

Reply:
I mean tab spacing in previous comments. You had something like that:
`.main {
display: flex;
  align-items: center;
  -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
          box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
outline: 2px solid red
 } `
 
 I asked you to make lite like this:
 `.main {
   display: flex;
   align-items: center;
   box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
   -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
   outline: 2px solid red 
   }`
   
   But not delete space after `:` symbol.
   Thanks you for your patient, sorry for misunderstanding.

Reply:
I remove it again

Reply:
Can you ping me in Telegram?